### ISSUE TITLE
AR not found

### ISSUE THREAD

**Issue by Khodour** [immersive-web/webxr-samples#107]
I am trying to test "hit test" but i am getting message on start button "AR not found" 

is it no longer working ?

**Comment by maletiger0404** [comment]
I have the same issue.

**Comment by bialpio** [comment]
Can you share some more information about the device, OS version and browser version that you're using? Do you see any errors in the console when navigating to the sample?

**Comment by maletiger0404** [comment]
My phone is iPhone X 14.4.2
The iOS phones doesn't support the WebXR?

**Comment by Khodour** [comment]
i have iPhone Xs too and its not working on safari and Chrome same issue AR not found , i think its require AR device no!

**Comment by maletiger0404** [comment]
yes, But look at this [AR supported device list](https://developers.google.com/ar/devices#ios).
There is the iPhone X on the list.
This list is for the unsupported devices?

**Comment by AdaRoseCannon** [comment]
iOS devices only have Webkit, the engine which Safari uses, all other browsers are just a wrapper around Webkit rather than using their own browser engines. Apple doesn't allow other browsers on iOS. Webkit doesn't yet support WebXR. It is currently in development so should hopefully land soon. 

https://webkit.org/status/#specification-webxr

**Comment by maletiger0404** [comment]
How can I implement the ground surface detection in AR using WebXR for Android and iOS both?

**Comment by bialpio** [comment]
This is not possible since there is currently no browser that supports WebXR on iOS.

**Comment by herrpedro** [comment]
please try to open your webxr app in this browser
https://itunes.apple.com/us/app/webxr-viewer/id1295998056?mt=8

**Comment by BismarckKL** [comment]
I am using Mi10 but it also show AR not Found at sample hit text with anchors, any ideas? Thanks

**Comment by ErickWendel** [comment]
> please try to open your webxr app in this browser https://itunes.apple.com/us/app/webxr-viewer/id1295998056?mt=8

worked on this one!
